plugins/DBSize/plugin.rb

```diff
@@ -0,0 +1,39 @@
+require 'sequel'
+require_relative '../../lib/BasePlugin/plugin.rb'
+require_relative '../../environment.rb'
+require_relative '../../models.rb'
+
+# Disk Sizes data plugin
+class DBSize < BasePlugin
+  def initialize
+    @@name = 'DBSize'
```

Comment:
In the other plugins iirc these variables also need to be set *before* the initialize block. I don't remember why I had to do that but just make sure this won't bite us later.